### PR TITLE
Added layout and day buttons for schedule

### PR DESCRIPTION
**Pull request by aymanelamyaghri**

### CODE REVIEW

**Review by hamzaremmal — CHANGES_REQUESTED** [review]
Tests  are missing, lets work on that so we can get this PR merged :-)

File: app/src/main/java/com/github/versus/MyAdapter.java
Comment (ktotam1) [inline app/src/main/java/com/github/versus/MyAdapter.java:1]: what does MyAdapter do? not clear from class name

Comment (aymanelamyaghri) [reply app/src/main/java/com/github/versus/MyAdapter.java:1]: Fixed this issue !

**Review by hamzaremmal — APPROVED** [review]
let's get this merge :-)

**Pull request merged**